css-text/text-indent/text-indent-hanging-001.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+<meta charset="utf-8">
+<title>CSS Text: text-indent: hanging</title>
+<meta name="flags" content="font">
+<meta name="assert" id="assert" content="text-indent: hanging makes all lines except the first line as indented">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+<link rel="author" title="hiroshi takase" href="mailto:takase@est.co.jp">
+<link rel="stylesheet" href="../../fonts/metadata_styles.css"/>
+<style>
+    @font-face {
+        font-family: 'webfont';
+        src: url('../../../fonts/ahem-extra/AHEM_default.TTF') format('opentype');
+        font-weight: normal;
+        font-style: normal;
+    }
+    #test, #ref {
+        position:absolute; top:0;
+        font-family: webfont;
+    }
+    #test {
+        width: 20em;
+        color: green;
+    }
+    #ref {
+        width: 15em;
+        padding-left: 5em;
+        text-indent: -5em;
+        color: red;
+        z-index: -100;
+    }
+/* the CSS above is not part of the test */
+    #test {
+        text-indent: 5em hanging;
+    }
+</style>
+</head>
+<body>
+
+<p class="instructions">Test passes if there is no red visible on the page.</p>
+
+<div style="position:relative">
+<div id="test">XXXX pppp XXXX pppp XXXX pppp XXXX pppp XXXX</div>

```

Comment:
I don't think this test works. 
Ideally, the green would show up in the gaps at the start of the 2nd & 3rd lines.
Perhaps the ref text would be something like:
```
<div id="ref"><span class="red">XXXX</span> XXXX XXXX XXXX<br><span class="green">XXXX</span> XXXX XXXX XXXX<br><span class="green">XXXX</span> XXXX</div>
```
And the test would be that the green block is visible, but no red. The colour of the test text should not be green.